embedded-graphics/src/pixelcolor/rgb_color.rs

```diff
@@ -295,4 +299,11 @@ mod tests {
         let color: Bgr555 = RawU16::from(0xFFFF).into();
         assert_eq!(RawU16::from(color).into_inner(), 0x7FFF);
     }
+
+    #[test]
+    fn convert_to_raw() {
+        let color = Rgb888::new(0xaa, 0xbb, 0xcc);
```

Comment:
I only added `Rgb888` here as I'm banking on the macro impls all doing the same/right thing internally.